### DISCUSSION TITLE
background color for articles

### DISCUSSION THREAD

**Discussion by krippels** [templaza/astroid-framework#364] [Q&A]
maybe this is easy, but I can't find it. I want to change the background color for articles in blog-view. where can I do that?
here ist the homepage: https://www.krippels.com/tiroler/index.php

thanks for your help
peter

**Comment by deltapapa01** [comment]
Hi Peter,
on which page exactly do you mean with blog view?
On the home page, the yellow headline in the light design is very difficult to read, is that what you mean ?

WBR from germany :-)

**Reply by krippels** [reply]
Hallo danke für deine Hilfe - ja genau auf der Startseite sollte der Hintergrund der Artikel in rot sein. Ich finde das einfach nicht :(

**Comment by deltapapa01** [comment]
Meinst du so?
------------------------------
Do you mean like this?
![image](https://github.com/templaza/astroid-framework/assets/19833064/b24fd7e7-932b-487f-9e54-452776368d48)

**Reply by krippels** [reply]
yes exactly and the regular text should be white

**Comment by deltapapa01** [comment]
Hi, sorry to get back to you so late, but now I have to ask: How do you put the boxes with the articles on the home page? Are these modules or is that an extra component?

For the color: I would give the bootstrap cards their own class and then format them via CSS, that's probably the easiest and best way. But for that I need to know what creates these Cards (see question above)
WBR,
deltapapa

DEU:
Hi, sorry das ich mich erst so spät wieder melde, aber jetzt muss ich nochmal fragen: Wie stellst du die Boxen mit den Artikeln auf der Startseite da? Sind das Module oder ist das eine extra Komponente?

Für die Farbe: Ich würde den Bootstrap Cards eine eigene Klasse geben und dann via CSS formatieren, dass ist wahrscheinlich der einfachste und beste Weg. Dafür muss ich jedoch wissen, was diese Cards erzeugt (siehe Frage oben)

![image](https://github.com/templaza/astroid-framework/assets/19833064/729bd335-3290-49ab-a68b-4fc79860f692)

Mit besten Grüßen,
deltapapa

**Reply by krippels** [reply]
The three boxes are featured articles. no leading article and 3 intro article set in the blog layout from the menu-item.
<img width="1546" alt="image" src="https://github.com/templaza/astroid-framework/assets/141207206/59973e17-8bca-4232-a27e-08f6d4275f83">

**Reply by krippels** [reply]
<img width="1612" alt="image" src="https://github.com/templaza/astroid-framework/assets/141207206/5c7731a6-0a9c-4598-9107-4634c591fff0">

**Reply by krippels** [reply]
I fought I can set this anywhere in the astroid template colors or layout. but I cant't find it.

**Comment by deltapapa01** [comment]
Uh-huh, okay. I would do things a little differently there. The featured articles are a Joomla component that you can control via overrides. You can see that if you switch the template once, for example, then the boxes look completely different.
I assume that on your page the content of the 3 boxes does not change constantly, right?
In this case I would create a new column in the layout manager of the Astroid, with 3 positions. You could also take the position bottom(x), and delete the bottom4, but I have found when testing, that there is a small bug (@sonvnn : I'll report later).
If you have the 3 positions, I would represent there the module Newsflash, then you can dadrinnen the appropriate articles publish. Or you create custom modules, which then link to the corresponding articles (so I always do).
And now you can change the colors of the font, the background etc. in the layout manager of the Astroid without problems:
Example: https://p603133.mittwaldserver.info/cms/

----
Aha, okay. Da würde ich etwas anders vorgehen. Die featured articles sind eine Joomla Komponente, die du über Overrides steuern kannst. Das siehst du, wenn du z.B. das Template einmal umschaltest, dann sehen die Boxen völlig anders aus.
Ich gehe davon aus, das auf deiner Seite der Inhalt der 3 Boxen sich nicht ständig ändert, oder?
In diesem Fall würde ich im Layout Manager vom Astroid eine neue Spalte Anlegen, mit 3 Positionen. Du könntest auch die Position bottom(x) nehmen, und die bottom4 löschen, jedoch habe ich beim testen festgestellt, das es da einen kleinen Bug gibt (@Sonny: melde ich später)
Wenn du die 3 Positionen hast, würde ich da das Modul Newsflash darstellen, dann kannst du dadrinnen die entsprechenden Artikel veröffentlichen. Oder du erstellst Custom Module, die dann auf die entsprechenden Artikel verlinken (so mache ich das immer).
Und jezt kannst du im Layout Manager vom Astroid die Farben der Schrift, des Hintergrundes etc. ohne Probleme ändern:
Beispiel: https://p603133.mittwaldserver.info/cms/

Bilder:
![image](https://github.com/templaza/astroid-framework/assets/19833064/92d2d688-07eb-4f5c-acff-98730c624cc9)
![image](https://github.com/templaza/astroid-framework/assets/19833064/e3e99023-c6f7-41c6-94d0-876f1ac14312)
![image](https://github.com/templaza/astroid-framework/assets/19833064/0103bce1-d093-42ac-938e-b4613e9babf1)
![image](https://github.com/templaza/astroid-framework/assets/19833064/6464924d-34ba-43e2-b891-3d05c158b413)

**Reply by krippels** [reply]
wow - thank you so much. this is great!
